src/main/java/io/github/jhipster/config/apidoc/customizer/JHipsterSwaggerCustomizer.java

```diff
@@ -74,8 +72,6 @@ public void customize(Docket docket) {
             .protocols(new HashSet<>(Arrays.asList(properties.getProtocols())))
             .apiInfo(apiInfo)
             .forCodeGeneration(true)
-            .directModelSubstitute(ByteBuffer.class, String.class)
```

Comment:
Who changing this to beans ? It was simpler this way !

Reply:
Fixed